src/Sulu/Component/HttpKernel/SuluKernel.php

```diff
@@ -78,6 +78,7 @@ protected function getKernelParameters()
             parent::getKernelParameters(),
             [
                 'sulu.context' => $this->getContext(),
+                'sulu.preview' => false,
```

Comment:
Alternatively we could also remove that line, and also ask if the parameter exists at all in the `SuluDocumentManagerExtension`.